taipy/gui/_renderers/factory.py

```diff
@@ -597,6 +597,19 @@ class _Factory:
             ]
         )
         ._set_propagate(),
+        "ProgressBar": lambda gui, control_type, attrs: _Builder(
```

Comment:
you also need to declare the component and it's default property at the top of the file
I propose `value`
and then add a call to `.set_value_and_default(var_type=PropertyType.dynamic_number)`

Reply:
But, there's not default param which we are using to render the Progress? Do I've to add the `value` param and its call `.set_value_and_default(var_type=PropertyType.dynamic_number)`?
Actually, I'm not able to completely understand that what it does if we add this, could you please guide?

Reply:
We want the developper to be able to set the value of the progress from the python backend and that's what that dynamic number typed `value` property is for.

Reply:
> We want the developper to be able to set the value of the progress from the python backend and that's what that dynamic number typed `value` property is for.

Actually, I've a doubt in this. Are you saying that if the developer wants to set the progress value according to their needs, then for this we're adding the `value` param, right? 
Suppose, if the developer wants the progress to stop at 90%, then the entered value will be 90, have I understood it correctly?

Reply:
not stop but set

Reply:
what I mean is that if a dev wants the progress to change value he will have to set this value (unless it is indeterminate)